apps/gallery/style/gallery.css

```diff
@@ -446,6 +446,10 @@ a.button.disabled {
   pointer-events: none;
 }
 
+a#thumbnails-share-button.disabled, a#thumbnails-delete-button.disabled{
+  pointer-events: all;
+}
+
```

Comment:
IMO instead of adding the new declaration,  pointer-events property can be set inside #thumbnails-delete-button and #thumbnails-share-button selector at Line 483 and Line 491
https://github.com/mozilla-b2g/gaia/blob/master/apps/gallery/style/gallery.css#L483